tests/test_api_audit.py

```diff
@@ -55,7 +55,7 @@ def test_02_get_allowed_audit_realm(self):
             json_response = json.loads(res.data.decode('utf8'))
             self.assertTrue(json_response.get("result").get("status"), res)
             self.assertEqual(json_response.get("result").get("value").get(
-                "count"), 9)
+                "count"), 8)
```

Comment:
Just to be sure: I guess this is because we have one less ``POST /auth`` entry now?

Reply:
correct